src/truenas_pylibzfs.c

```diff
@@ -121,7 +129,18 @@ PyInit_truenas_pylibzfs(void)
 		return (NULL);
 	}
 
-	add_constants(mpylibzfs);
+	constants = PyModule_Create(&truenas_pylibzfs_constants);
+	if (constants == NULL) {
+		Py_DECREF(mpylibzfs);
+		return (NULL);
+	}
+
+	add_constants(constants);
+
+	if (PyModule_AddObject(mpylibzfs, "constants", constants) < 0) {
+		Py_DECREF(mpylibzfs);
```

Comment:
In case of failure, do we need `Py_DECREF(constants)` here?

Reply:
Yes :)